ostd/src/arch/x86/cpu/tdx.rs

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use tdx_guest::{handle_virtual_exception, tdcall, TdxTrapFrame};
+
+use crate::cpu::{RawGeneralRegs, UserContext};
+
+pub(crate) fn handle_virtualization_exception(user_context: &mut UserContext) {
```

Comment:
The names of the two functions, `handle_virtual_exception` and `handle_virtualization_exception`, are too much alike. From names, a reader cannot get their relationship easily. Importing `handle_virtual_exception` with a prefix to make their relationship more obvious.

```rust
use tdx_guest::{handle_virtual_exception as do_handle_virtual_exception, tdcall, TdxTrapFrame};

pub(crate) fn handle_virtual_exception(user_context: &mut UserContext) {
    let ve_info = tdcall::get_veinfo().expect("#VE handler: fail to get VE info\n");
    let mut generalrags_wrapper = GeneralRegsWrapper(*user_context.general_regs_mut());
    do_handle_virtual_exception(&mut generalrags_wrapper, &ve_info);
    *user_context.general_regs_mut() = generalrags_wrapper.0;
}
```